containers/images/content-server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM quay.io/foreman/pulp
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release && \
+    yum -y install python-pulp-streamer && \
+		yum clean all
+
+RUN rm -f /etc/httpd/conf.d/pulp.conf /etc/httpd/conf.d/pulp_puppet.conf
```

Comment:
Each Dockerfile instruction like `RUN` creates a new layer and from what I understood - avoiding adding more layers to the image is a best practice.

Thus, maybe you can merge these next lines into:

```
RUN rm -f /etc/httpd/conf.d/pulp.conf /etc/httpd/conf.d/pulp_puppet.conf /etc/pki/tls/certs/localhost.crt /etc/pki/tls/private/localhost.key && \
    ln -s /var/lib/pulp/published /var/www/pub && \
    ln -s /etc/pki/pulp/certs/pulp.crt /etc/pki/tls/certs/localhost.crt && \
    ln -s /etc/pki/pulp/private/pulp.key /etc/pki/tls/private/localhost.key && \
    chown apache:apache /var/lib/pulp && \
    sed -i 's/127.0.0.1/squid/g' /etc/httpd/conf.d/pulp_streamer.conf
```
Unless there are caching or other concerns that made you decide to do that.

Reply:
It does reduce layers, however, I think it makes readability worse. I think we can explore image sizes across the board after the initial round to weigh size reduction vs. understandbility.